tests/responses/tests.py

```diff
@@ -136,6 +138,20 @@ def test_repr(self):
         expected = '<HttpResponse status_code=201, "text/html; charset=utf-8">'
         self.assertEqual(repr(response), expected)
 
+        response = HttpResponse(status=204)
+        del response['Content-Type']
+        expected = '<HttpResponse status_code=204>'
+        self.assertEqual(repr(response), expected)
+
+        response = HttpResponseNotAllowed(('GET', 'POST'))
```

Comment:
tests for the `repr`s of `HttpResponseNotAllowed` and `HttpResponseRedirect` already exist in `httpwrappers/tests`, please move these tests there